### PR TITLE
Allow revert to default key

### LINKED ISSUE #45: Handle situation to revert to F4C account key
If a user defines their own account key that is currently put in place of the Fold for Covid key in `global_prefs_override.xml`, but we do not handle the situation to revert back to the Fold for Covid key if the user removes their own.

### LINKED ISSUE #49: Setting ACCOUNT_KEY give error message
Setting the env var `ACCOUNT_KEY` give the error:
`Invalid or missing account key.  To fix, remove and add this project.`

### PR DESCRIPTION
**Pull request by chrisys**
Change-type: patch
Signed-off-by: Chris Crocker-White <chriscw@balena.io>

### CODE REVIEW

**Comment by chrisys** [comment]
There are instances where changing the key and reverting can cause `06-May-2020 16:45:26 [Rosetta@home] Invalid or missing account key.  To fix, remove and add this project.`. Need to debug this more

File: boinc-client/start-utils.sh [hunk 1]
```diff
@@ -24,3 +24,33 @@ validate_ram_settings() {
...
+    local new_key="$ACCOUNT_KEY"
+  fi
+
+  # is the key changing?
+  if [[ ! "$current_key" = "$new_key" ]]; then
+    echo 'Account key change detected - purging'
+    rm -rf /usr/app/boinc/*
+
+    # set the new key with a fresh version of template file
+    cp "$account_template_file_path" "$account_file_path"
```
Comment (tmigone) [inline boinc-client/start-utils.sh:49]: Shouldn't this be copied back into `/usr/app/boinc`?

Comment (tmigone) [reply boinc-client/start-utils.sh:49]: NVM, just saw the `cd /usr/app/boinc` on the other script. Still a weird mix of absolute and relative paths :P Could be improved later!

Comment (chrisys) [reply boinc-client/start-utils.sh:49]: good point, I did put the absolute path here as I didn't want to leave `rm -rf *` in a script!

**Review by tmigone — APPROVED** [review]

**Comment by tmigone** [comment]
@balena-ci retest

**Pull request merged**